env/pom.xml

```diff
@@ -23,17 +23,17 @@
             <artifactId>java-api</artifactId>
             <version>${project.version}</version>
         </dependency>
-        <!-- Required only in case of multiple xsuaa bindings -->
         <dependency>
-            <groupId>com.sap.cloud.security.xsuaa</groupId>
-            <artifactId>token-client</artifactId>
-            <version>${project.version}</version>
-            <scope>provided</scope>
+            <groupId>com.sap.cloud.environment.servicebinding</groupId>
+            <artifactId>java-sap-service-operator</artifactId>
```

Comment:
Just to get that straight: This module (the `com.sap.cloud.security:env`) is supposed to **only** support K8s based applications, right?

Reply:
yes, as of now